airflow/contrib/operators/gcp_compute_operator.py

```diff
@@ -193,9 +193,7 @@ class GceSetMachineTypeOperator(GceBaseOperator):
         Defaults to False.
     :type validate_body: bool
     """
-    # [START gce_instance_set_machine_type_template_fields]
```

Comment:
Why should we remove those comments? They are used to embed code snippets into documentation - they should not be removed IMHO. Same about few more comments like that below.

Reply:
You are right. @mik-laj said that it was deprecated so I thought that it could be removed. My bad... I have already reverted those changes.